core/storage/email/gae_models.py

```diff
@@ -377,8 +407,17 @@ def has_reference_to_user_id(cls, user_id):
 
     @classmethod
     def create(
-            cls, instance_id, recipient_ids, sender_id, sender_email,
-            intent, subject, html_body, sent_datetime):
+            cls,
+            instance_id, # type: Text
+            recipient_ids, # type: Optional[List[Text]]
```

Comment:
Why is this optional? That seems weird to me conceptually... won't emails always need a recipient or are there cases in which they do not? If the latter, what are those cases?

Reply:
Yes I think I'm wrong here. There is no test here which checks for None. Changed it to `List[str]. Thanks!